src/components/ArticleIndexPage/ArticleIndexPage.js

```diff
@@ -77,9 +88,8 @@ const ArticleIndexPage = props => {
       activeCategory={category}
     >
       <Content>
-        <Heading>
-          {title} <Count>{articles.length}</Count>
-        </Heading>
+        <Heading>{title}</Heading>
+        <Count>{articles.length} articles</Count>
```

Comment:
Small thing, but in theory and for consistency the `articles` text should live in the UI texts:

https://github.com/sharetribe/flex-docs/blob/master/docs/coding-conventions.md#use-ui-texts

The original idea was to have a place that enabled adding translations for the UI later on. That might still be far fetched as all the article content is still in English, but it's still a good practise to track down all the content.